geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/membership/gms/messenger/JGroupsMessengerJUnitTest.java

```diff
@@ -484,6 +498,38 @@ public void testBigMessageIsFragmentedWhenBroadcast() throws Exception {
     doTestBigMessageIsFragmented(false, true);
   }
 
+  public static class ByteHolder implements DataSerializableFixedID {
```

Comment:
Maybe it would make sense to move this class to the bottom of the test class so it isn't between test methods?  I don't feel super strongly about it, just an idea.

Reply:
okay